addons/pos_event/models/event_event.py

```diff
@@ -6,6 +6,29 @@ class EventEvent(models.Model):
     _inherit = ['event.event', 'pos.load.mixin']
 
     image_1024 = fields.Image("PoS Image", max_width=1024, max_height=1024)
+    pos_orders_price_total = fields.Integer('PoS orders price total', compute='_compute_pos_orders_price_total', groups='point_of_sale.group_pos_user')
+
+    @api.depends('company_id.currency_id', 'registration_ids.pos_order_line_id.price_subtotal_incl',
+                 'registration_ids.pos_order_line_id.currency_id', 'registration_ids.pos_order_line_id.order_id.state')
+    def _compute_pos_orders_price_total(self):
+        date_now = fields.Datetime.now()
+        event_subtotals = self.env['pos.order.line']._read_group(
+            [('event_ticket_id.event_id', 'in', self.ids), ('price_subtotal_incl', '!=', 0), ('order_id.state', 'in', ['paid', 'done'])],
```

Comment:
```suggestion
            [('event_ticket_id.event_id', 'in', self.ids), ('order_id.state', 'in', ['paid', 'done'])],
```
IMO, not needed as some event ticket prices are zero! So, when I click on the button of `**PoS Sales**` It should redirect me to all orders which are related to the event! 
Also check for company domain if needed!